http_service/bugbug_http/worker.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
```

Comment:
Can you make a first commit where you just move the file and one where you apply the modifications? This way maybe git will be able to tell that it isn't a new file and we won't lose the history

Reply:
I just did that in 52f4eb4d125145e7a70bcdf094115e78938c6854 (moving files) and then 24fe16099a82458d3e3ddd7208093fad191cfe9b (patching files), but the result is the same.

Reply:
If you want to keep track of the history, you should merge without squashing